runtime/parachains/src/inclusion/mod.rs

```diff
@@ -194,13 +207,15 @@ pub mod pallet {
 		+ shared::Config
 		+ paras::Config
 		+ dmp::Config
-		+ ump::Config
 		+ hrmp::Config
 		+ configuration::Config
 	{
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 		type DisputesHandler: disputes::DisputesHandler<Self::BlockNumber>;
 		type RewardValidators: RewardValidators;
+
+		/// The system message queue.
+		type MessageQueue: EnqueueMessage<ParaId>;
```

Comment:
It's unclear from either the typename or the doc comment that this is intended to be used for UMP

Reply:
It's not UMP specific. In principle it could be used for things other than UMP.

Reply:
Added a comment.